doc(release) Release notes for web 24.04.7 / open tickets 24.04.1 (#3612)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.04/releases/centreon-os.mdx

## Centreon Web

### 24.04.7

Release date: `September 30, 2024`

<details open>
  <summary>Enhancements</summary>

- [API] Added an endpoint that automatically deploys the services brought by the host template.
- [Authentication] Rescheduled the task of deleting expired tokens.
- [Resource Status] Users can now filter by host name in view by host mode.

</details>

<details>
  <summary>Bug fixes</summary>

- [Administration] Fixed access to the Add token button.
- [Authentication] Fixed an issue with the ‘X509, MultiFactor’ authentication method.

> - [Authentication] Fixed incorrect URL redirection issue with OpenId Connect.

Reviewer: ```suggestion
- [Authentication] Fixed incorrect URL redirection issue with OpenID Connect.
```

- [Authorization] Fixed an issue on ACLs reloading.
- [API] Added missing configuration change logs.
- [API] Fixed access control for monitoring servers configuration endpoints.
- [API] Fixed ACL for meta-service endpoint.

> - [CLAPI] Fixed the display of version.

Reviewer: ```suggestion
- [CLAPI] The -V command now displays the version of Centreon with no unnecessary message.
```

- [Configuration] Fixed an issue on mass change which enabled disabled services.

> - [Configuration] Fixed an issue on central server ID in Gorgone configuration.

Reviewer: ```suggestion
- [Configuration] The central server’s ID is now correctly configured in the Gorgone configuration
```

> - [Configuration] Fixed an issue on getting metrics on MetaService form.

Reviewer: ```suggestion
- [Configuration] Fixed an issue with the list of available metrics in the metaservices form.
```

- [Configuration] The correct value for the notification activation option is now written to the Engine configuration file.

> - [Dataset] Fixed filtering when searching a resource.

Reviewer: ```suggestion
- [Dataset] Fixed filtering when searching for a resource.
```

- [Dataset] Handled special characters in metrics.

> - [Dashboards] Fixed host search for resource status widgets.

Reviewer: ```suggestion
- [Dashboards] Fixed the search for hosts in the resource status widget.
```

- [Graph] Fixed an error that occurred when editing curves.
- [Graph] You can now select a metric in the curve and virtual metric configuration forms.
- [Install] The 24.04.0 update wizard can now be rerun if required.

> - [Packaging] Fixed invalid permissions on oudated token cron.

Reviewer: ```suggestion
- [Packaging] Fixed invalid permissions on the cron that deletes obsolete tokens.
```

> - [Performance] Fixed a query performance issue with some specific datas in resource status.

Reviewer: ```suggestion
- [Performance] Fixed a query performance issue with some specific data on the Resource status page.
```

> - [Resource Status] Fixed an issue that occured when a non-admin user was searching for resources by host categories.

Reviewer: ```suggestion
- [Resource Status] Fixed an issue that occurred when a non-admin user was searching for resources by host categories.
```

> - [Resource Status] Fixed an issue with "Force Active Checks" option which was ignored in some cases.

Reviewer: ```suggestion
- [Resource Status] Fixed an issue with the "Force Active Checks" option, which was ignored in some cases.
```

- [Resource Status] Fixed the responsivity and visibility of "display by" views.

> - [Status Chart widget] Fixed issue with the list of resources displayed when hovering the charts.

Reviewer: ```suggestion
- [Status Chart widget] Fixed an issue with the list of resources displayed when hovering the charts.
```

> - [Status Chart widget] Filter on selected resources is now handled correctly.

Reviewer: ```suggestion
- [Status Chart widget] Filtering on selected resources is now handled correctly.
```

> - [Status Grid widget] Filter on selected resources is now handled correctly in the condensed view.

Reviewer: ```suggestion
- [Status Grid widget] Filtering on selected resources is now handled correctly in the condensed view.
```

## Centreon Open Tickets

### 24.04.1

Release date: `September 30, 2024`

<details open>
  <summary>Enhancements</summary>

> - [OpenTickets] Fixed redirection to monitoring pages according to the use deprecated pages option.

Reviewer: ```suggestion
- [OpenTickets] Fixed redirection to monitoring pages according to the "use deprecated pages" option.
```
